src/components/FormCard/FormCard.jsx

```diff
@@ -11,6 +11,8 @@ import { selectTechnologies, selectJobs, selectSeniority } from 'App/ListData/se
 import { fetchListData } from 'App/ListData/slice';
 
 import { getListByName } from 'Services/salaries';
+import { englishInfo, seniorityInfo } from 'Constants';

```

Comment:
listSenority is the same that seniorityInfo.
english_level is the same that englishInfo.

Reply:
export const listSenority = ['Junior', 'Mid-Junior', 'Mid', 'Mid-Senior', 'Senior'];
export const ListEnglish = ['A1', 'A2', 'B1', 'B2', 'C1', 'C2'];

They're not the same shape. listSenority and ListEnglish are not being used. I will delete them from Constants